src/main/java/org/sonarlint/intellij/actions/SonarLintToolWindow.java

```diff
@@ -469,5 +473,20 @@ private Content getTaintVulnerabilitiesContent() {
   @Override
   public void bindingChanged(@Nullable ProjectBinding previousBinding, @Nullable ProjectBinding newBinding) {
     runOnUiThread(project, this::refreshViews);
+
+    showSharedConfigurationNotification(project, "Project successfully bound with \"" +
```

Comment:
You could leverage the triple quotes `"""`

Reply:
I prefer this approach, IMO it is more readable

Reply:
I think it's better with a text block; you don't have to escape the quotes inside the text block and it removes all the plus signs on each line. Overall, I don't see a reason not to use it. Also, we started using this text block in other parts of the code, and as I mentioned in another comment, we should stay consistent in the codebase.